src/components/common/TicketEditor/TicketEditor.tsx

```diff
@@ -279,15 +286,30 @@ const TicketEditor = observer(
                 >
                   Copy
                 </ActionButton>
+                <ActionButton
+                  color="primary"
+                  onClick={handlePreview}
+                  disabled={isCopying}
+                  data-testid="copy-description-btn"
+                >
+                  {isPreview ? 'Preview' : 'Edit'}
```

Comment:
If it is in preview state, the Edit button should be what is visible and when it's in Edit phase the button should say Edit.